src/Nest/XPack/Security/Role/PutRole/IApplicationPrivileges.cs

```diff
@@ -53,6 +53,9 @@ public class ApplicationPrivilegesDescriptor : DescriptorPromiseBase<Application
 			Assign(a => a.AddIfNotNull(selector?.Invoke(new ApplicationPrivilegesDescriptor<T>())));
 	}
 
+
+	//TODO this should not be generic
```

Comment:
Should this TODO be fixed before merging?

Reply:
Will be addressable in `7.x`